WebAppForSolocoProject/Services/OwnerData.cs

```diff
@@ -15,8 +16,8 @@ public IEnumerable<Owner> GetOwners()
             Owners = ReadFromConfig();
             Owners = CheckForChildOwners(Owners);
             RemoveStringThatIsNotPath(Owners);
-            ConvertStringToPaths(Owners);
-            return Owners;
+            ConvertConfigToFolderPathPair(Owners);
+            return Owners.OrderBy(o => o.Name);
         }
 
         private List<Owner> ReadFromConfig()
```

Comment:
too long methode, split it. Maybe create separate class for config stuffs where you will put methode for parsing config and in OwnerData just call them